dotnet/src/Agents/OpenAI/AnnotationContent.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using Azure.AI.OpenAI.Assistants;
+
+namespace Microsoft.SemanticKernel.Agents.OpenAI;
+
+/// <summary>
+/// Content type to support message annotations.
+/// </summary>
+public class AnnotationContent : KernelContent
```

Comment:
Should this be in SK.Abstractions?

Reply:
@RogerBarreto it may be interesting to you.

There two aspects here:
1. It should live to the abstraction in a form that can be reused by other components i.e AI connectors, agents, etc.
2. If it does not live in the abstraction project it'll be a challenge at the moment to have it serializable because it need to be registered for polymorphic serialization on the bese `KernelContent` class.

Reply:
Makes sense, will move.